apps/music/js/app.js

```diff
@@ -175,8 +175,11 @@ viewStack.addEventListener('change', (evt) => {
 
   document.body.dataset.activeViewUrl = viewUrl;
 
-  playerButton.hidden = viewUrl === VIEWS.PLAYER.URL || !isPlaying;
-  activityDoneButton.hidden = viewUrl !== VIEWS.PLAYER.URL || !activity;
+  var showingPlayer = viewUrl === VIEWS.PLAYER.URL;
+  playerButton.hidden = showingPlayer || !isPlaying;
+  activityDoneButton.hidden = !showingPlayer || !activity;
+  client.method('enableNFC', showingPlayer);
```

Comment:
Errrr.. Disregard this. What you're doing here is probably preferred since it allows us to separate the endpoint into the SW later.